src/Gui/Application.h

```diff
@@ -30,6 +30,10 @@
 
 #include <App/Application.h>
 
+#if QT_VERSION >= QT_VERSION_CHECK(6,0,0) && QT_VERSION < QT_VERSION_CHECK(6,7,4)

```

Comment:
dow e have confirmation that this is fixed in qt 6.7.4?

Reply:
Honestly not yet. It is P1 in Qt, but it is just educated wish. I'm tracking this, so probably NP to set no limit now.

Reply:
In that case it's better to only check qt > 6 and we'll add the upper limit when we have a confirmed fix in qt

Reply:
Done